src/mui/button/SaveButton.js

```diff
@@ -6,20 +6,23 @@ import ContentSave from 'material-ui/svg-icons/content/save';
 import CircularProgress from 'material-ui/CircularProgress';
 import translate from '../../i18n/translate';
 
-class SaveButton extends Component {
+export class SaveButton extends Component {
 
     handleClick = (e) => {
         if (this.props.saving) {
             // prevent double submission
             e.preventDefault();
+        } else if (!this.props.submitOnEnter && this.props.handleSubmit) {
+            this.props.handleSubmit();
```

Comment:
Great idea! To do this I need to know the name of the form, which I'm not sure how to get. I've seen `record-form` and `record-form-<index>` namings as well. 
Now a see that probably the indexed naming is the new one although I don't see naming on the create forms, only on the edit ones. 
- Should there be explicit names defined for the create forms?
- Should I pass down the form name to this component as I don't see how I could get it in any other way?